.circleci/config.yml

```diff
@@ -396,8 +394,11 @@ jobs:
             EOL
             git add package.json yarn.lock
             git commit -m "[CI] Update dependencies" || exit 0
-            yarn test -u storyshots
-            git commit -a -m "[CI] Update snapshots" || true
+
+            # 2023-07-03 (ns): disabling as storyshots are disabled
+            # yarn test -u storyshots
```

Comment:
I wonder if this should still run the tests after updating deps, or perhaps that'll get run elsewhere in the branch too?